internal/stack/compose.go

```diff
@@ -220,7 +220,7 @@ func newServiceStatus(description *docker.ContainerDescription) (*ServiceStatus,
 		Status:  description.State.Status,
 		Version: getVersionFromDockerImage(description.Config.Image),
 	}
-	if description.State.Status == "running" {
+	if description.State.Status == "running" && description.State.Health != nil {
```

Comment:
It could be kept in case of running and no health status is reported some default string (e.g. "-"), something like this (I could not add a suggestion from unmodified lines):

```go
  healthStatus := "-"
  if description.State.Health != nil {
    healthStatus = description.State.Health.Status
  }
  if description.State.Status == "running" {
    service.Status = fmt.Sprintf("%v (%v)", service.Status, healthStatus)
  }
```

Thereby, that service is still shown in the output table, but with no health status (yet). WDYT ?

Reply:
Umm, with current code it will show `running`, with your proposal, it will show `running (-)`, I think I prefer just `running` in the case we don't have more info.

Reply:
Well, it may be ok to print something like `running (unknown health)`.

Reply:
Pushed like this, let me know what you think.

Reply:
Yes, I prefer that option, printing "unknown health" is better than just showing "-" from my first suggestion. It gives more information to the user :+1: 
In this way, if the service is running, it is always shown in the output even without health status.